client/webserver/site/src/js/locales.ts

```diff
@@ -111,6 +111,21 @@ export const ID_WALLET_SYNC_FINISHING_UP = 'WALLET_SYNC_FINISHING_UP'
 export const ID_CONNECT_WALLET_ERR_MSG = 'CONNECTING_WALLET_ERR_MSG'
 export const ID_REFUND_IMMINENT = 'REFUND_IMMINENT'
 export const ID_REFUND_WILL_HAPPEN_AFTER = 'REFUND_WILL_HAPPEN_AFTER'
+export const ID_AVAILABLE_C = 'ID_AVAILABLE_C'
+export const ID_LOCKED_C = 'ID_LOCKED_C'
+export const ID_IMMATURE_C = 'ID_IMMATURE_C'
```

Comment:
What is the `_C` suffix? Can you choose something more informative?

Reply:
> What is the _C suffix? Can you choose something more informative?

`Capitalized`. Maybe I should use the full term?

Reply:
It just wasn't obvious. If you don't want to write out `_CAPITALIZED`, you can use `_TITLE`